tests/unittest/_torch/models/test_minimax_m3.py

```
@@ -0,0 +1,1458 @@
+_CHECKPOINT_PATH_ENV = "MINIMAX_M3_CHECKPOINT_PATH"
+_DEFAULT_CHECKPOINT_PATH = "/home/scratch.fredw_sw/workspace/hidden_trail/minimax-m3-preview_vv1"
+
+
+def _checkpoint_path() -> str:
+    return os.environ.get(_CHECKPOINT_PATH_ENV, _DEFAULT_CHECKPOINT_PATH)
+
+
+def _has_cuda() -> bool:
+    try:
+        import torch
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)